docs/central-backup.rst

```diff
@@ -7,94 +7,43 @@ Having a data backup strategy is a critical part of running a web service like O
+You will additionally need to have a backup of Enketo data to be able to restore existing Web Form links. The strict minimums required to do this are Enketo's Redis store and the keys generated in the Enketo configuration. **In general, we recommend making a full system backup.**
 
-If you don't already have a full system backup in place and don't want to set up your own database backup, Central provides a managed backup system to Google Drive.
+If you don't already have a full system backup in place and don't want to set up your own database backup, Central provides an API endpoint to download a backup of the database.
 
-.. _central-managed-backups:
+.. _central-direct-backups:
 
-Managed backups / Direct backups via API
-----------------------------------------
+Direct Backups via API
+----------------------
 
 .. warning::
 
-  Managed backups and `Direct backups via API <https://odkcentral.docs.apiary.io/#reference/system-endpoints/direct-backup/data-document>`_ include all of your collected data but do **NOT** include sufficient information to re-establish the same Web Form links. If you use :ref:`Public Links <central-submissions-public-link>` for broad surveying or share :ref:`links to Web Forms <central-submissions-direct>` through another system, we strongly recommend also making a full system backup.
+  `Direct Backups via API <https://odkcentral.docs.apiary.io/#reference/system-endpoints/direct-backup>`_ include all of your collected data but do **NOT** include sufficient information to re-establish the same Web Form links. If you use :ref:`Public Links <central-submissions-public-link>` for broad surveying or share :ref:`links to Web Forms <central-submissions-direct>` through another system, we strongly recommend also making a full system backup.
 
-  If you only use web forms for previews or for making submissions directly from Central, managed backups are sufficient. You can regenerate previews by uploading the same form with a new :ref:`form version <central-forms-updates>`. 
+  If you only use Web Forms for previews or for making Submissions directly from Central, Direct Backups are sufficient. You can regenerate previews by uploading the same Form with a new :ref:`Form version <central-forms-updates>`.
 
-ODK Central features an optional off-site backup system that backs up all Central data (but as detailed above, not Web Form configurations).
+The Central API offers an HTTP endpoint to perform an immediate backup of the database and download the result to your computer. We call this type of backup a Direct Backup and recommend it only for specific use cases. For each Direct Backup, we extract all your data (including user accounts, Forms, and Submissions), then encrypt it with an optional passphrase. The encrypted backup will be returned in the HTTP response. As detailed above, the backup will not include Web Form configurations.
```

Comment:
"encrypt it with an optional passphrase" is probably not the most clear. I wanted to convey:

- The result is always encrypted.
- You may specify a passphrase.
- But you don't have to specify a passphrase.

My instinct is that we shouldn't delve too much into encryption and the passphrase here. What I didn't want to imply is that encryption without a passphrase provides any additional security. Users who are interested in Direct Backups will navigate to the API docs, which offer more details on these subjects.